docs/resources/tag_association.md

```diff
@@ -81,7 +81,7 @@ resource "snowflake_tag_association" "column_association" {
 ### Required
 
 - `object_identifier` (Block List, Min: 1) Specifies the object identifier for the tag association. (see [below for nested schema](#nestedblock--object_identifier))
-- `object_type` (String) Specifies the type of object to add a tag to. ex: 'ACCOUNT', 'COLUMN', 'DATABASE', etc. For more information: https://docs.snowflake.com/en/user-guide/object-tagging.html#supported-objects
+- `object_type` (String) Specifies the type of object to add a tag. Allowed object types: [                              ].
```

Comment:
empty?

Reply:
Not sure why it was this way, but after moving the validations it generated correctly (most likely it was something with `init` function and moving it to the other package that created some kind of dependency).